### ISSUE TITLE
ol-cesium@2.2.0 (from 2) breaks webpack build

### ISSUE THREAD

**Issue by tlepkowski** [openlayers/ol-cesium#629]
Hello,

Holding everything else constant, when I go from this 
`"dependencies": {
    "ol": "^5.1.3",
    "cesium": "^1.47.0",
    "ol-cesium": "^2.1.0"
  },`

to this:
` "dependencies": {
    "ol": "^5.1.3",
    "cesium": "^1.47.0",
    "ol-cesium": "^2.2.0"
  },`

in my package.json file, via  npm install --save ol-cesium@2.2.0

`ReferenceError: ol is not defined[Learn More] olcesium.js:5585 ` error message in my Firefox console.

and in the Chrome console: `Uncaught ReferenceError: ol is not defined
    at Object.ol/geom/Point.js (olcesium.js:5585)
    at i (olcesium.js:29)
    at Module../src/olcs/OLCesium.js (olcesium.js:2584)
    at i (olcesium.js:29)
    at Module../src/index.library.js (olcesium.js:134)
    at i (olcesium.js:29)
    at ./src/goog/asserts.js (olcesium.js:93)
    at Object.<anonymous> (olcesium.js:10)
    at i (bootstrap:19)
    at Module.<anonymous> (obj.js:60)`

It looks like I need to expose the whole ol namespace in my main.js file (the app's entry point).

**Comment by gberaudo** [comment]
How do you use OL-Cesium?
If you build OL-Cesium together with your application code it should work (see how it is done in the OL-Cesium examples).

**Comment by tlepkowski** [comment]
No, not along with application code.  The application code does not lend
itself to this kind of building.  I build ol-cesium it as a separate unit
because I have N apps that use my build.  It is kept neatly separate.
